spec/models/notification_spec.rb

```diff
@@ -491,20 +491,12 @@
 
   describe "#send_new_badge_achievement_notification" do
     it "enqueues a new badge achievement job" do
-      assert_enqueued_with(job: Notifications::NewBadgeAchievementJob, args: [badge_achievement.id]) do
+      sidekiq_assert_enqueued_with(job: Notifications::NewBadgeAchievementWorker, args: [badge_achievement.id]) do
         described_class.send_new_badge_achievement_notification(badge_achievement)
       end
     end
   end
 
-  describe "#send_new_badge_notification (deprecated)" do
```

Comment:
Removed this test as it was part of an intermittent alias, see https://github.com/thepracticaldev/dev.to/pull/5358/files#r363057123